docs/application/vscode-ext/web.md

```diff
@@ -12,64 +12,37 @@ Before installing VS Code Extension for Tizen Web, check [Prerequisites of Visua
 
 To install VS Code Extension for Tizen Web from the Visual Studio Code Marketplace, follow these steps:
 
-1. In the VS Code at the bottom of the **Activity Bar** click **Extensions** (![extension icon](media/extensions.png)).
+1. In the Activity Bar of VS Code, click **Extensions** ![extension icon](media/extensions.png).
 
    ![Visual Studio Code Marketplace](media/vscode_marketplace.png)
 
-2. In the **Extensions** view, enter **Tizen Web** as search term in the **Search Extensions in Marketplace** field.
-3. Locate **Visual Studio Code Extension for Tizen Web** in the search results and click **Install**.
-4. After the installation is complete, click **Reload** to activate the extension.
-5. Once the extension activates, install the SDK to effectively use the extension.
-6. Open **Command Palette** and enter **Create Wizard Project**. The confirmation message dialog appears:
-
-   ![Visual Studio Code Marketplace](media/tizen_web1.png)
-
-   > **Note**
-   >
-   > Extension is a convenient way to create a project. It asks whether you want to use an existing Tizen baseline SDK installation or install a new one.
-
-  - To perform a fresh installation, click **YES**.
-
-    To perform a fresh installation and to begin the Tizen Baseline SDK installation, click YES. Else, click No, if you have already done the installation.
+2. In the **EXTENSIONS** view that appears, click the search box at the top and type **Tizen Web**.
+3. In the **Tizen Web** search result that appears, click **Install**.
+4. After the installation is complete, to activate the extension, click **Reload**.
 
-    > **Note**  
-    >
-    > In case you click **No**, it is assumed that the installation is done and the project is already created.
+    After the extension is active, **Tizen Web** installation is complete.
 
 ## Install using VSIX file
```

Comment:
@	DohyungLim, please change file to File.

Install using VSIX File

Reply:
done.